modules/sdk-coin-atom/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './atom';
+export * from './tatom';
```

Comment:
Needs new `register` method added to other coin modules.

Reply:
Already working on including it in my next PR, ty!